packages/valory/skills/mech_interact_abci/skill.yaml

```diff
@@ -182,6 +182,11 @@ models:
         priority_mech_address: '0x0000000000000000000000000000000000000000'
         response_timeout: 300
       nvm_balance_tracker_address: '0x7D686bD1fD3CFF6E45a40165154D61043af7D67c'
+      did_registry_address: '0x0000000000000000000000000000000000000000'
+      agreement_store_manager_address: '0x0000000000000000000000000000000000000000'
+      lock_payment_condition_address: '0x0000000000000000000000000000000000000000'
+      transfer_nft_condition_address: '0x0000000000000000000000000000000000000000'
+      escrow_payment_condition_address: '0x0000000000000000000000000000000000000000'
```

Comment:
@KahanMajmudar could you provide the default values?

Reply:
Gnosis:
AgreementStoreManager: 0x4Cd9b03bEB7D68bC397B64521DF0A272bE94a4Df
DIDRegistry: 0xCB0A331cB1F57E01FF0FA2d664f2F100081cbc3b
EscrowPaymentCondition: 0x31B2D187d674C9ACBD2b25f6EDce3d2Db2B7f446
LockPaymentCondition: 0x2749DDEd394196835199471027713773736bffF2
TransferNFTCondition: 0x659fCA7436936e9fe8383831b65B8B442eFc8Ea8

Base:
AgreementStoreManager: 0x173CFb11baa0Cf18FDA698cF82AEf6181D84B845
DIDRegistry: 0x5FD091093152403BEE33a5c4Db60721Fc513985D
EscrowPaymentCondition: 0xA33f6149563CfEC51C9e7961A3FB5DdF5F9D5B68
LockPaymentCondition: 0x7d1b782A347234d1442e57721444B778D5B2E6B7
TransferNFTCondition: 0x17a49d0942b987ebDE9D6400A045159bd3936541

Reply:
Are these always going to be the same per chain? If yes, we could make them part of `CHAIN_TO_NVM_CONFIG`. Otherwise, I'll set the Gnosis ones as the default.

Reply:
Yes same for given chains